consul/src/main/java/com/linecorp/armeria/internal/consul/ConsulDefaults.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.internal.consul;
+
+/**
+ * Consul related constant values.
+ */
+public final class ConsulDefaults {
```

Comment:
I think we can remove this class because:
- `DEFAULT_CONNECT_TIMEOUT_MS` is unused.
- `DEFAULT_SESSION_TIMEOUT_MS` is unused.
- `DEFAULT_CHECK_INTERVAL` is only used in `ConsulUpdatingListener`. This can move to it.